packages/react-native-reanimated/src/createAnimatedComponent/PropsFilter.tsx

```diff
@@ -87,7 +86,7 @@ export class PropsFilter implements IPropsFilter {
         if (component._isFirstRender) {
           props[key] = value.value;
         }
-      } else if (key !== 'onGestureHandlerStateChange' || !isChromeDebugger()) {
+      } else {
```

Comment:
`isChromeDebugger()` always returns false, so the condition always evaluates to true.